examples/turbulent_advection-diffusion.jl

```diff
@@ -191,7 +194,7 @@ contour!(p, x, y, Array(ψ[1]'), levels = -0.15:-0.3:-1.5, lw=2, c=:grey, ls=:da
 
```

Comment:
Suggest 

```julia
# Create time series for the concentration and streamfunction in the bottom layer, `layer = 2`.
file = jldopen(output.path)

iterations = parse.(Int, keys(file["snapshots/t"]))
t = [file["snapshots/t/$i"] for i ∈ iterations]

layer = 2

c = [file["snapshots/concentration/$i"][:, :, layer] for i ∈ iterations]
ψ = [file["snapshots/streamfunction/$i"][:, :, layer] for i ∈ iterations]
nothing # hide
```

to tidy up lines 160-170?

Reply:
Yeap!